README.md

```diff
@@ -2,35 +2,6 @@
 
 Avo audit helps you detect issues in your raw analytics event tables by querying a sample of the raw events and comparing them against each other.
 This allows you to find discrepancies between events and properties that should look the same, but don’t.
```

Comment:
Should we update the intro text here to better describe the current functionality?